### PR TITLE
Fix amendments to check method in sympy.stats.crv_types

### LINKED ISSUE #16842: Make amendments to check method in sympy.stats.crv_types
#### Changes to be made
Due to the merging of https://github.com/sympy/sympy/pull/16765 , some changes are needed to be made to `sympy.stats.crv_types` in the way, `check` method uses `_value_check` function. 
The task is to rewrite the `check` method in `sympy.stats.crv_types` for distributions which uses `_value_check` in the following manner,
```
_value_check(<cond1> <bool_op1> <cond2> <bool_op2> <cond3> <bool_op3> . . . <bool_opn> <condn>, msg)
```
`condi` is  the ith condition, `bool_opi` is the ith bool operator and msg is message. n > 0 or at least one bool operator should be there to make the change to `_value_check`.
The above `_value_check` can be rewritten as,
```
_value_check((<cond1>, <cond2>, <cond3>, . . ., <condn>), msg)
```
For detailed discussion threads, see the links in references.

#### Other comments/references
1. Related PRs - https://github.com/sympy/sympy/pull/16765, https://github.com/sympy/sympy/pull/16571
2. Current Code - [sympy.stats.crv_types](https://github.com/sympy/sympy/blob/master/sympy/stats/crv_types.py)
3. Priority - Low
4. Difficulty - Easy

### PR DESCRIPTION
**Pull request by bytes-as**
#### References to other Issues or PRs

Fixes #16842 

#### Brief description of what is fixed or changed
Due to merging of #16765, the way the check method uses _value_check
is better to be changed in some cases as shown in the example below

#### Other comments
earlier :
```
_value_check(d1 > 0 and d1.is_integer, \
        "Degrees of freedom d1 must be positive integer.")
_value_check(d2 > 0 and d2.is_integer, \
        "Degrees of freedom d2 must be positive integer.")
```
as of now :
```
_value_check((d1 > 0, d1.is_integer), \
    "Degrees of freedom d1 must be positive integer.")
_value_check((d2 > 0, d2.is_integer), \
    "Degrees of freedom d2 must be positive integer.")
```
#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by czgdp1807** [comment]
@solus9 Please add the missing attributes and checks which https://github.com/sympy/sympy/pull/16571 was trying to do. Many of the distributions in `sympy.stats.crv_types`(master) don't have the checks and attributes. Make about 3 to 4 PRs to complete the `sympy.stats.crv_types`.

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -1128,9 +1128,9 @@ class FDistributionDistribution(SingleContinuousDistribution):
 
     @staticmethod
     def check(d1, d2):
-        _value_check(d1 > 0 and d1.is_integer, \
+        _value_check((d1 > 0, d1.is_integer), \
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py]: I think `\` isn't needed in `()`.

File: sympy/stats/crv_types.py [hunk 2]
```diff
@@ -1128,9 +1128,9 @@ class FDistributionDistribution(SingleContinuousDistribution):
...
             "Degrees of freedom d1 must be positive integer.")
-        _value_check(d2 > 0 and d2.is_integer, \
+        _value_check((d2 > 0, d2.is_integer), \
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py]: Same for `\`.

Comment (solus9) [reply sympy/stats/crv_types.py]: yeah, that's right
thanks

**Pull request merged**